src/main/scala/nl.knaw.dans.easy.deposit/docs/DDM.scala

```diff
@@ -69,19 +69,20 @@ object DDM extends SchemedXml with DebugEnhancedLogging {
         { dm.spatialPoints.withNonEmpty.map(point => <dcx-gml:spatial srsName={ point.srsName }>{ details(point) }</dcx-gml:spatial>) }
         { dm.spatialBoxes.withNonEmpty.map(point => <dcx-gml:spatial>{ details(point) }</dcx-gml:spatial>) }
         { dm.license.withNonEmpty.map(src => <dcterms:license xsi:type={ src.scheme.nonBlankOrNull }>{ src.value.nonBlankOrEmpty }</dcterms:license>) }
+        { dm.languagesOfFiles.withNonEmpty.flatMap(lang => lang.value.map(value => <dcterms:language xsi:type ={ lang.scheme.nonBlankOrNull  }>{ value }</dcterms:language>)) }
```

Comment:
As mentioned in https://github.com/DANS-KNAW/easy-deposit-api/pull/158#discussion_r295351969, probably you should use `lang.key` here? Please discuss with @lindareijnhoudt first!

Reply:
So KeyLess values like "Goerees" or other non ISO639-2 values are lost? I'm quite lost here now frankly. Ill will discuss this with @lindareijnhoudt